include/openssl/bn.h

```diff
@@ -974,6 +974,12 @@ OPENSSL_EXPORT int BN_bn2binpad(const BIGNUM *in, uint8_t *out, int len);
 // BN_secure_new calls |BN_new|.
 OPENSSL_EXPORT BIGNUM *BN_secure_new(void);
 
+#define BN_FLG_MALLOCED 0x01
+#define BN_FLG_STATIC_DATA 0x02
+#define BN_FLG_CONSTTIME   0x04
```

Comment:
This can imply that code makes false assumptions about the run-time behaviour of AWS-LC. How is that mitigated?